app/docker/networks/network.service.ts

```diff
@@ -0,0 +1,39 @@
+import axios, { parseAxiosError } from '@/portainer/services/axios';
+import { EnvironmentId } from '@/portainer/environments/types';
+
+import { NetworkId, DockerNetwork } from './types';
+
+export async function getNetwork(
+  environmentId: EnvironmentId,
+  networkId: NetworkId
+) {
+  try {
+    const { data: network } = await axios.get<DockerNetwork>(
+      buildUrl(environmentId, networkId)
+    );
+    return network;
+  } catch (e) {
+    throw parseAxiosError(e as Error, 'Unable to retrieve network details');
+  }
+}
+
+export async function removeNetwork(
```

Comment:
```suggestion
export async function deleteNetwork(
```

remove is used mostly when removing a relation (i.e remove user from team), while delete says the object is removed (or deleted) from the db

Reply:
updated